portal/templates/eligibility/eligibility-criteria.njk

```diff
@@ -42,11 +42,11 @@
 
         <fieldset class="govuk-fieldset">
 
-          <legend class="govuk-fieldset__legend govuk-fieldset__legend">
+          <legend class="govuk-fieldset__legend govuk-fieldset__legend" tabindex=0 aria-label="Eligibility criterion {{ criterion.id }}">
             Eligibility criterion {{ criterion.id }}
           </legend>
 
-          <p class="govuk-body govuk-hint">{{ criterion.description }}</p>
+          <p class="govuk-body govuk-hint" tabindex=0 aria-label="{{ criterion.description }}">{{ criterion.description }}</p>
```

Comment:
`tabindex=0` won't work properly for more than one element. Do we need it at all?

Reply:
it seems to work, and yes, its the only way to get all the bits to be recognised by narrator